src/Common/src/Interop/Windows/kernel32/Interop.CriticalSection.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct CRITICAL_SECTION
+        {
+            IntPtr _debugInfo;
+            long _lockCount;
```

Comment:
`LONG` in Windows C++ does not map to `long` in C#. LONG in Windows C++ is 32-bit integer, and thus it should be int in C#.